Core/Core_c.php

```diff
@@ -660,3 +660,90 @@ public static function create(object $referent): WeakReference {}
      */
     public function get(): ?object {}
 }
+
+/**
+ * Stringable interface marks classes as available for serialization
+ * in a string.
+ *
+ * @since 8.0.0
+ */
+interface Stringable {
+    /**
+     * Magic method {@see https://www.php.net/manual/ru/language.oop5.magic.php}
+     * called during serialization to string.
+     *
+     * @return string
+     */
+    public function __toString(): string;
+}
+
+/**
+ * @since 8.0.0
+ */
+// TODO Uncomment after PHP 8.0 release:
+// @@Attribute(Attribute::TARGET_CLASS)
+final class Attribute {
+    /**
+     * Marks that attribute declaration is allowed only in classes.
+     *
+     * @var int
+     */
+    public const TARGET_CLASS = 1;
+
+    /**
+     * Marks that attribute declaration is allowed only in functions.
+     *
+     * @var int
+     */
+    public const TARGET_FUNCTION = 1 << 1;
+
+    /**
+     * Marks that attribute declaration is allowed only in class methods.
+     *
+     * @var int
+     */
+    public const TARGET_METHOD = 1 << 2;
+
+    /**
+     * Marks that attribute declaration is allowed only in class properties.
+     *
+     * @var int
+     */
+    public const TARGET_PROPERTY = 1 << 3;
+
+    /**
+     * Marks that attribute declaration is allowed only in class constants.
+     *
+     * @var int
+     */
+    public const TARGET_CLASS_CONSTANT = 1 << 4;
+
+    /**
+     * Marks that attribute declaration is allowed only in function or method parameters.
+     *
+     * @var int
+     */
+    public const TARGET_PARAMETER = 1 << 5;
+
+    /**
+     * Marks that attribute declaration is allowed anywhere.
+     *
+     * @var int
+     */
+    public const TARGET_ALL = (1 << 6) - 1;
+
+    /**
+     * Notes that an attribute declaration in the same place is
+     * allowed multiple times.
+     *
+     * @var int
+     */
+    public const IS_REPEATABLE = 1 << 10;
+
+    /**
+     * @param int $flags
```

Comment:
PhpDoc is redundant and can be safely removed

Reply:
In the PSR it is written that the code should be uniform.

Therefore, I put down docblocks for all types, even where it is not needed in order to maintain this uniformity.

Is removal definitely required?

Reply:
https://github.com/JetBrains/phpstorm-stubs/pull/857/commits/484f9fcc81fc9b3bc0afdd19b5878d0bb748769d